packages/flutter/test/cupertino/button_test.dart

```diff
@@ -205,21 +205,53 @@ void main() {
     await tester.pump();
     FadeTransition transition = tester.firstWidget(find.byType(FadeTransition));
 
+    await tester.pump(const Duration(milliseconds: 25));
```

Comment:
Can you make these 50ms increments? 25 is a bit granular for this.

Reply:
sure 🚀